api/src/org/labkey/api/util/element/Input.java

```diff
@@ -580,12 +594,15 @@ protected void doStyles(Appendable sb) throws IOException
         }
     }
 
-    protected void doInputEvents(Appendable sb) throws IOException
+    protected void doInputEvents(String id, Appendable sb) throws IOException
```

Comment:
Can we remove the unused `Appendable` parameter?
```suggestion
    protected void doInputEvents(String id) throws IOException
```